docs/docs/write-pages.md

```diff
@@ -59,70 +59,71 @@ The dynamic files that are created are (all under the `.cache` directory).
 
 ## async-requires.js
 
 ---
 
-`async-requires.js` is very similar to `sync-requires.js`, in that it is a dynamically generated JavaScript file. The difference is that it is written to be used for code splitting via webpack. So, instead of using `require` with the component's path, it uses `import` and adds a `webpackChunkName` hint so that we can eventually link the componentChunkName to its resulting file (more info in [Code Splitting](/docs/how-code-splitting-works/) docs). `components` is a function, so that it can be lazily initialized.
+O `async-requires.js` é muito semelhante ao `sync-requires.js`, pois é um arquivo JavaScript gerado dinamicamente. A diferença é que ele foi escrito para ser usado na divisão de código via webpack. Portanto, ao invés de usar `require` com o caminho do componente, ele usa `import` e adiciona uma dica `webpackChunkName` para que possamos eventualmente vincular o componentChunkName ao arquivo resultante (mais informações em [Code Splitting](/docs/how-code-splitting-works/) docs). `components` é uma função, para que possa ser inicializada lentamente.
 
-`async-requires.js` also exports a `data` function that imports `data.json` ([see below](/docs/write-pages/#datajson))
+`async-requires.js` também exporta uma função `data` que importa `data.json` ([see below](/docs/write-pages/#datajson))
 
-An example of async-requires is:
+Um exemplo de requisição assíncrona é:
 
 ```javascript
 exports.components = {
   "component---src-blog-2-js": () =>
     import(
       "/home/site/src/blog/2.js" /* webpackChunkName: "component---src-blog-2-js" */
     ),
-  // more components
-}
+  // mais componentes
+};
 
-exports.data = () => import("/home/site/.cache/data.json")
+exports.data = () => import("/home/site/.cache/data.json");
 ```
 
-Remember, `sync-requires.js` is used during [Page HTML Generation](/docs/html-generation/). And `async-requires.js` is used by [Building the JavaScript App](/docs/production-app/).
+Lembre-se de que, `sync-requires.js` é usado durante [Page HTML Generation](/docs/html-generation/). E o `async-requires.js` é usado por [Building the JavaScript App](/docs/production-app/).
 
 ## data.json
 
-This is a generated json file. It contains the entire `pages.json` contents ([as above](/docs/write-pages/#pagesjson)), and the entire redux `jsonDataPaths` which was created at the end of the [Query Execution](/docs/query-execution/#save-query-results-to-redux-and-disk) stage. So, it looks like:
+Este é um arquivo json gerado. Ele contém todo o conteúdo do `pages.json` ([como acima](/docs/write-pages/#pagesjson)), e todo o redux `jsonDataPaths` que foi criado no final do [Execução de Consulta](/docs/query-execution/#save-query-results-to-redux-and-disk) estágio. Então, isso parece com:
```

Comment:
```suggestion
Este é um arquivo json gerado. Ele contém todo o conteúdo do `pages.json` ([como acima](/docs/write-pages/#pagesjson)), e todo o redux `jsonDataPaths` que foi criado no final da [Execução de Consulta](/docs/query-execution/#save-query-results-to-redux-and-disk) estágio. Então, isso parece com:
```

Reply:
Então, aqui eu usei `do` porque eu acredito que a referência seja para a palavra `estágio` que vem depois dos colchetes e parênteses. Você acha melhor referenciar ao que vem dentro do colchetes?

Reply:
Dentro dos colchetes, pois vai virar texto corrido, os colchetes são do markdown para adicionar links.

Reply:
Sugiro:
`que foi criado no final da [execução de consulta](/docs/query-execution/#save-query-results-to-redux-and-disk) do estágio.`

que resultara em:
que foi criado no final da [execução de consulta](/docs/query-execution/#save-query-results-to-redux-and-disk) do estágio.

Reply:
Boa!